src/libraries/System.Private.CoreLib/src/System/MathF.cs

```diff
@@ -471,5 +477,47 @@ public static unsafe float Truncate(float x)
             ModF(x, &x);
             return x;
         }
+
+        public static float ScaleB(float x, int n)
```

Comment:
Just noting, since this PR was opened, AMD open sourced their own `libm` implementation which includes `scaleb`: https://github.com/amd/aocl-libm-ose/blob/7193ff6fdf72ecd68df4e0e19262da84f1d95a3e/src/ref/scalbnf.c

It notably has a different algorithm and error handling logic (unnecessary in the C# implementation) but it might be worth comparing in the future to see which is better.